src/smc-webapp/editor.coffee

```diff
@@ -1545,6 +1545,12 @@ class CodeMirrorEditor extends FileEditor
         # show the codemirror editors, resizing as needed
         @_show_codemirror_editors(cm_height, width)
 
+        # changes the width on resize change
+        width_resize = () =>
+            width = @element.find(".salvus-editor-codemirror-chat-column").offset().left
+            @_show_codemirror_editors(cm_height, width)
+        $(window).resize(width_resize)
```

Comment:
Wow. Just wow.  Every single time the editor is shown you're creating a new function and assigning a new event handler.   It's never freed.  This is some seriously leaky code.